.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Bug Report
+about: Create a bug report for rust-libp2p.
+---
+
+<!-- Thank you for filing a bug report! -->
+
+<!-- For questions please use the rust-libp2p GitHub Discussions forum.-->
```

Comment:
Could this be linked like discuss.libp2p.io in `config.yaml`?

Reply:
Good idea!